app/templates/view_suppliers.html

```diff
@@ -24,11 +24,7 @@
   {% endwith %}
 
   <div class="page-section">
-    {% if current_user.has_role("admin-ccs-sourcing") %}
-      {% include "_view_suppliers_agreements.html" %}
```

Comment:
We can delete `_view_suppliers_agreements.html` now as it's not used - or can we? It's the only place where CCS Sourcing users can download documents for old frameworks (G7 and earlier - these can't be viewed as an embedded file, unlike later framework docs). They won't need the 'Upload countersigned agreement' link for the old frameworks as they should be done and dusted now.

A possible solution could be to amend the 'View agreements' link on the supplier details page's frameworks table for G7 and earlier, to point to the 'Download signed agreement' url that was previously on the search results.